roles/monitoring-client/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Retrieve our hostname
+  set_fact:
+    host_name: "{{ ansible_hostname }}"
```

Comment:
do you use this anywhere?
why not simply consume ansible_hostname instead of host_name ?